pyroSAR/drivers.py

```diff
@@ -1516,16 +1525,15 @@ def __init__(self, scene):
                        r'(?P<relative_orbit>[0-9]{5})_' \
                        r'(?P<absolute_orbit>[0-9]{5})_' \
                        r'(?P<counter>[0-9]{4,})\.' \
-                       r'(?P<satellite_ID>[EN][12])' \
-                       r'(?P<extension>(?:\.zip|\.tar\.gz|))$'
-        
+                       r'(?P<satellite_ID>[EN][12])'
         self.pattern_pid = r'(?P<sat_id>(?:SAR|ASA))_' \
                            r'(?P<image_mode>(?:IM(?:S|P|G|M|_)|AP(?:S|P|G|M|_)|WV(?:I|S|W|_)|WS(?:M|S|_)))_' \
                            r'(?P<processing_level>[012B][CP])'
         
         self.scene = os.path.realpath(scene)
         
-        self.examine()
+        # Only a file to examine
+        self.file = self.scene
```

Comment:
Please rewrite it to make sure that `self.file` directly points to the file if the scene is zipped as well as after calling `self.unpack`:
```python
if re.search('.[EN][12]$', self.scene):
    self.file = self.scene
else:
    self.examine()
```